Recrutify/Recrutify.Services/Services/CandidateService.cs

```diff
@@ -132,5 +134,19 @@ public Task<bool> ExistsAsync(Guid id)
         {
             return _candidateRepository.ExistsAsync(id);
         }
+
+        public async Task BulkCreateTestFeedbacksAsync(CreateBullFeedbackTestDTO testResult, Guid userId)
```

Comment:
testresult rename to bulkCreateTestFeedbackDto

Reply:
+